File: week2/day3/factorio.py

# - Create a function called `factorio`
#   that returns it's input's factorial


p = int(input("Tell me the number until you want to factorial the numbers please: "))

def factorio(p):
    x = 1
    y = 1
    while x < p:
        x += 1
        y *=x
    print(y)

factorio(p)

File: week8/day3/factorio.js
'use strict';
// - Create a function called `factorio`
//   that returns it's input's factorial

function factorio(number) {
    var factorial = 1;
    for (var i = 1; i < (number + 1); i++) {
        factorial = factorial * i;
    }
    return factorial;
}
console.log(factorio(5));
